Automattic/wp-calypso
# Redirect to WooCommerce installation from Store dashboard if WooCommerce is not installed

Fixes #50354

#### Changes proposed in this Pull Request

* Redirects to WooCommerce installation (`/woocommerce-installation`) if WooCommerce is not installed
* Removes everything except for the removal notice from the dashboard

#### Testing instructions



1. Run branch
2. With a non-Business/non-eCommerce plan site, verify that `/store/SITE_SLUG` redirects to `/home/SITE_SLUG`
3. With a non-Atomic Business plan site, verify that `/store/SITE_SLUG` redirects to `/home/SITE_SLUG`
4. With an Atomic Business plan site (install Hello Dolly plugin) without WooCommerce, verify that `/store/SITE_SLUG` redirects to `/woocommerce-installation/SITE_SLUG`

<img width="1275" alt="Screen Shot 2021-03-04 at 16 25 23" src="https://user-images.githubusercontent.com/2098816/110032607-54263d00-7d06-11eb-94c4-a9f6feaeac8c.png">

5. With an Atomic Business plan site with WooCommerce, verify that `/store/SITE_SLUG` shows removal notice and sidebar links go to wp-admin (sidebar links only will show if store has finished address setup or has products -- this is existing sidebar behavior)
6. With an Atomic eCommerce plan site with WooCommerce, verify that `/store/SITE_SLUG` shows removal notice and sidebar links go to wp-admin (sidebar links only will show if store has finished address setup or has products -- this is existing sidebar behavior)

<img width="1275" alt="Screen Shot 2021-03-04 at 16 24 58" src="https://user-images.githubusercontent.com/2098816/110032568-48d31180-7d06-11eb-8cf7-b7fa99a274e6.png">
